tests/LogTargetTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace yiiunit\extensions\debug;
+
+use yii\debug\LogTarget;
+use yii\debug\Module;
+
+class LogTargetTest extends TestCase
+{
+    public function testGetRequestTime()
+    {
+        $logTarget = new LogTarget(new Module('debug'));
+        $actual = $this->invoke($logTarget, 'getRequestTime');
+        $this->assertTrue(is_float($actual));
```

Comment:
Actually @dynasource is right because `REQUEST_TIME_FLOAT` may be unavailable in super-weird server configurations and there's a fallback for that case.